docs/usage/configuration-options.md

```diff
@@ -1232,6 +1232,13 @@ Typically you would encrypt it and put it inside the `encrypted` object.
 
 See [Private npm module support](https://docs.renovatebot.com/getting-started/private-packages) for details on how this is used.
 
+## npmrcMerge
+
+This option exists to provide flexibility about whether `npmrc` strings in config should override `.npmrc` files in the repo, or be merged with them.
```

Comment:
Can you explain in the text why you might need/want to use this?

It's not clear to me what the possible benefit is to using/not using this option... 😉

Reply:
See changes